src/DestinationRepository.js

```diff
@@ -0,0 +1,9 @@
+import Destination from './Destination';
+
+class DestinationRepository {
+    constructor(destinationData) {
+			this.destinations = destinationData.map((destinationObj) => { return new Destination(destinationObj) });
+    }
```

Comment:
Oh nice you create the destination objects in here.  I wonder if me not doing it in here will cause me trouble!

Reply:
I don't think it'll cause you any problems with how you have it! Just a different way to do it :)